src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/StopTask.groovy

```diff
@@ -22,7 +22,11 @@ class StopTask extends AbstractServerTask {
     @TaskAction
     void stop() {
         if (new File("${project.buildDir}/wlp/").exists()) {
-            executeServerCommand(project, 'stop', buildLibertyMap(project))
+        	if (getServerDir(project).exists()) {
+            	executeServerCommand(project, 'stop', buildLibertyMap(project))
+        	} else {
+        		logger.error ('There is no server to stop!')
```

Comment:
Making a suggestion for both the new and old message. How about:

There is no server to stop.  The server has not been created.